### ISSUE TITLE
WYSIWYG ToolBar Keeps Focus

### ISSUE THREAD

**Issue by My-Random-Thoughts** [tinymce/tinymce#8705]
**What is the current behavior? Describe the bug**
Initially logged with BookStack: https://github.com/BookStackApp/BookStack/issues/4173

When clicking a sub-menu item on the toolbar, the focus immediately returns focus to the page to keep typing, but the sub-menu is still shown. Clicking on the editor page to remove the sub-menu and keep typing results in the toolbar stealing focus away from the editor.

**Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tiny.cloud](https://fiddle.tiny.cloud/) or similar.**
https://jsfiddle.net/ssddanbrown/0ye9f1g8/1/

    Add some text
    Click "..." in toolbar
    Click Any toolbar button
    Type some more (works)
    Click next to typed text
    Type some more (fails as toolbar as focus)
    Click again next to typed text
    Type some more (works again)

**What is the expected behavior?**
Once the button has been clicked the sub-menu should disappear and focus returned to the main editor

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**
6.3.1
Firefox 112.0.2
Linux Mint

**Comment by My-Random-Thoughts** [comment]
Still an issue

**Comment by tiny-ci-servers** [comment]
This issue is stale because it has been open 365 days with no activity. Please comment if you wish to keep this issue open or it will be closed in 30 days.

**Comment by My-Random-Thoughts** [comment]
Still an issue

**Comment by Phil-GOX** [comment]
I've also noticed this problem:
The toolbar group retains focus and the first icon remains selected after you click on the group icon and return to editing the text.

Browser : Edge 146.0.3856.84, Firefox 149.0

https://fiddle.tiny.cloud/fsjnGp2WRj/3

**Comment by My-Random-Thoughts** [comment]
I don't think this project is alive any more as nothing has happened for almost 3 years.
Shame really.

**Comment by Phil-GOX** [comment]
That's exactly what I observed! I commented more out of principle to confirm the bug, but without much hope of a resolution from them.
